src/main/java/org/broadinstitute/barclay/argparser/Argument.java

```diff
@@ -97,9 +97,46 @@
     int maxElements() default Integer.MAX_VALUE;
 
     /**
+     /**
      * This boolean determines if this annotation overrides a parent annotation. If that is the case then
      * the options of the parent annotation are overridden with this annotation.
      */
     boolean overridable() default false;
 
+    /**
+     * Hard lower bound on the allowed value for the annotated argument -- generates an exception if violated.
+     * Enforced only for numeric types whose values are explicitly specified on the command line.
+     *
+     * @return Hard lower bound on the allowed value for the annotated argument, or Double.NEGATIVE_INFINITY
+     *         if there is none.
+     */
+    double minValue() default Double.NEGATIVE_INFINITY;
```

Comment:
Are these bounds used in the parsing? If so, I would like to have this soon!! Thanks for it!